### ISSUE TITLE
[QUESTION]

### ISSUE THREAD

**Issue by mrcomac** [Larkinabout/fvtt-token-action-hud-core#189]
**Describe the question**
How to use the icon option available in the info?
Not sure if I'm understand correctly but I can see the info field supports text and icon https://github.com/Larkinabout/fvtt-token-action-hud-core/blob/main/templates/action.hbs#L15

when I try `const info1 = { icon: '<i class="fa fa-plus" aria-hidden="true"></i>' }` is doesn't work (the icon is just a placeholder for now).

When I use `const info1 = { text: 'test' }` it works.

The goal is to add up to 3 images right after the name of an item with its caracteristics, for example ranged/melee, slow, etc.

**Comment by Larkinabout** [comment]
You're using it correctly, it's just broken as `ActionHandler.#createAction` isn't passing on the property. I'll fix it for the next release.

Also, a reminder to me to add in the tooltip when using `icon`.

**Comment by Larkinabout** [comment]
Fixed in Token Action HUD Core 1.4.19.
